pleskdistup/common/src/files.py

```diff
@@ -59,25 +59,30 @@ def get_last_lines(filename: PathType, n: int) -> typing.List[str]:
         return f.readlines()[-n:]
 
 
-def backup_file(filename: str) -> None:
+def backup_file(filename: str, ext: str = ".bak") -> None:
     if os.path.exists(filename):
-        shutil.copy(filename, filename + ".bak")
+        shutil.copy(filename, filename + ext)
 
 
-def backup_exists(filename: str) -> bool:
-    return os.path.exists(filename + ".bak")
+def backup_exists(filename: str, ext: str = ".bak") -> bool:
+    return os.path.exists(filename + ext)
 
 
-def restore_file_from_backup(filename: str, remove_if_no_backup: bool = False) -> None:
-    if os.path.exists(filename + ".bak"):
-        shutil.move(filename + ".bak", filename)
+def restore_file_from_backup(filename: str, remove_if_no_backup: bool = False, ext: str = ".bak") -> None:
+    if os.path.exists(filename + ext):
+        shutil.move(filename + ext, filename)
     elif remove_if_no_backup and os.path.exists(filename):
         os.remove(filename)
 
 
-def remove_backup(filename: str) -> None:
-    if os.path.exists(filename + ".bak"):
-        os.remove(filename + ".bak")
+def remove_backup(filename: str, ext: str = ".bak", logf = None) -> None:
+    try:
+        if os.path.exists(filename + ext):
+            os.remove(filename + ext)
+    except Exception as ex:
+        if logf is None:
```

Comment:
I believe we could use `log.debug` here.

Reply:
I didn't want to break old behavior - throwing exception.

Reply:
I suggest adding some logging and then raising an exception. Or if you want to avoid an exception in your specific case, we could add a boolean parameter to configure the function's behavior.

Reply:
I don't think there is a benefit ATM, both w/exception (as before) and logging (my current need) is working as we want (when one needs a behavior change later they'll do it).

Reply:
The issue I have is with the argument's name. In my opinion, it does not explain why it should be used or how it will affect the function's behavio

Reply:
Added explicit `raise_exception` param